### PR TITLE
Added settings for the new tagger microservice.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: playbooks/de-deploy-services.yaml [hunk 1]
```diff
@@ -180,6 +180,24 @@
...
+  tags:
+    - services
+    - colored
+    - tagger
+  roles:
+    - role: util-notify-chat
+      msg: "Deploying tagger"
+    - role: de-deploy-service
+      deploy_use_color: "use_color|default(false)"
+      has_configs: "{% if use_consul_configs %}{{ false }}{% else %}{{ true }}{% endif %}"
```
Comment (psarando) [inline playbooks/de-deploy-services.yaml:196]: Too bad you can't just print the `true/false` value of `use_consul_configs` here 😖

**Review by psarando — APPROVED** [review]
LGTM :shipit:

File: playbooks/de-deploy-services.yaml (hunk 1 above)
Comment (psarando) [reply playbooks/de-deploy-services.yaml:196]: oops, except it's the opposite of the value....

**Comment by slr71** [comment]
Thanks for the review! 👍

**Pull request merged**